addons/website_event_track/static/src/interactions/website_event_track_proposal_form.js

```diff
@@ -0,0 +1,168 @@
+import { _t } from "@web/core/l10n/translation";
+import { Interaction } from "@web/public/interaction";
+import { post } from "@web/core/network/http_service";
+import { registry } from "@web/core/registry";
+import { renderToElement } from "@web/core/utils/render";
+import { scrollTo } from "@web_editor/js/common/scrolling";
+
+export class WebsiteEventTrackProposalForm extends Interaction {
+    static selector = ".o_website_event_track_proposal_form";
+
+    dynamicContent = {
+        ".o_wetrack_add_contact_information_checkbox" : { "t-on-click" : this.onAdvancedContactToggle },
+        "input[name='partner_name']" : { "t-on-input" : this.onPartnerNameInput },
+        ".o_wetrack_proposal_submit_button" : { "t-on-click.prevent.stop" : this.onProposalFormSubmit },
+        ".o_wetrack_contact_information" : { "t-att-class": () => ({
+            "d-none": !this.useAdvancedContact,
+            "o_wetrack_no_contact_mean_error": !this.hasContactMean,
+        }) },
+        ".o_wetrack_contact_mean": { "t-att-class": () => ({ "is-invalid": !this.hasContactMean }) },
```

Comment:
This attribute should be set to true in setup, otherwise the field is immediately marked as invalid.